pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -394,26 +428,49 @@ message Capability {
     repeated string drop_capabilities = 2;
 }
 
+// SecurityContext holds security configuration that will be applied to a container.
+message SecurityContext {
+    // Capabilities to add or drop.
+    optional Capability capabilities = 1;
+    // If set, run container in privileged mode.
+    optional bool privileged = 2;
+    // The configurations for the container's namespaces.
+    // This will be used only if the container uses namespace for isolation.
+    optional NamespaceOption namespace_options = 3;
+    // Optional SELinux context to be applied.
+    optional SELinuxOption selinux_options = 4;
+    // The UID to run the the container process.
+    // Defaults to user specified in image metadata if unspecified.
+    optional int64 run_as_user = 5;
+    // Indicates that the container must run as a non-root user.
+    // If true, the runtime should validate the image at runtime to ensure that it
+    // does not run as UID 0 (root) and fail to start the container if it does.
+    // If unset or false, no such validation will be performed.
+    optional bool run_as_non_root = 6;
```

Comment:
Can we move this check to kubelet? Kubelet can read the image metdata, verify `run_as_non_root`, and always set  `run_as_user` when running a container. Then we can remove `run_as_non_root` here.